replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/persistence/ReplicationPersistentStore.java

```diff
@@ -119,6 +130,7 @@ String getPersistenceType(Class<? extends AbstractPersistable> klass) {
   }
 
   public void save(AbstractPersistable persistable) {
+    persistable.setModified();
```

Comment:
✏️ Could be moved into the `toMap()` method such that handling of the modified timestamp be done internally inside the persistable item